update to container version 16

andreeaflorescu opened the pull request:
### Summary of the PR

This version adds some tooling for fuzzing and updated to Rust 1.63.0.

### Requirements

Before submitting your PR, please make sure you addressed the following
requirements:

- [x] All commits in this PR are signed (with `git commit -s`), and the commit
  message has max 60 characters for the summary and max 75 characters for each
  description line.
- [x] All added/changed functionality has a corresponding unit/integration
  test.
- [x] Any newly added `unsafe` code is properly documented.

lauralt reviewed (approved):
nit: we could also update the docs so that they point to the latest version published. It might actually be a good idea to replace the hardcoded values with v${LATEST} everywhere as we already do and explain [here](https://github.com/rust-vmm/rust-vmm-ci#running-the-tests-locally).

andreeaflorescu commented:
> nit: we could also update the docs so that they point to the latest version published. It might actually be a good idea to replace the hardcoded values with v${LATEST} everywhere as we already do and explain [here](https://github.com/rust-vmm/rust-vmm-ci#running-the-tests-locally).

I've updated the README as well and added a note that the version specified there might not be the latest one. We might forget to update it from time to time.
